### ISSUE TITLE
Support for multiple displays

### ISSUE THREAD

**Issue by l0drex** [zzag/plasma5-wallpapers-dynamic#68]
It would be nice if there was an option to have one wallpaper as the background for multiple displays.

**Comment by zzag** [comment]
Design-wise, this plugin is not suited for dynamic wallpapers spanning multiple displays. But PRs are always welcome. :-)

**Comment by l0drex** [comment]
Maybe the plugin could simply split the wallpaper and only uses the image for the display you are setting the image to. 
Let's say you have two displays, one on the left and one on the right. You go to the left screen, select the file, set the display amount to 2 and select the one on the left. The plugin then reads the file, splits it vertically and only uses the left part. Same for the right screen.
I don't know if this would work, but maybe the plugin could even read the monitor layout from the display configuration set in plasma. This would also make things easier if the displays are not on the same height.

Is this possible to do?

**Comment by zzag** [comment]
Yes, it's possible. The plugin has all necessary screen layout information, see https://doc.qt.io/qt-5/qml-qtquick-window-window.html#screen-prop.
